tds-itemselection-impl/src/test/java/msb/MsbAssessmentSelectionServiceTest.java

```diff
@@ -0,0 +1,268 @@
+        msbAssessmentSelectionService = new MsbAssessmentSelectionServiceImpl(itemSelectionDbLoader);
+    }
+
+    @After
+    public void tearDown() {}
+
+    private List<ItemCandidatesData> retrieveItemCandidatesData() {
+        ArrayList<ItemCandidatesData> itemCandidatesData = new ArrayList<>();
+        ItemCandidatesDataBuilder itemCandidateDataBuilder = new ItemCandidatesDataBuilder();
+
+        ItemCandidatesData candidate1 = itemCandidateDataBuilder
+                .build();
+        itemCandidatesData.add(candidate1);
+
+        ItemCandidatesData candidate2 = itemCandidateDataBuilder
+                .withSegmentKey("(SBAC_PT)SBAC-MSB-IRP-CAT-NoCalc-MATH-7-Summer-2015-2016")
+                .withSegmentID("SBAC-MSB-IRP-CAT-NoCalc-MATH-7")
+                .withSegmentPosition(2)
+                .build();
+        itemCandidatesData.add(candidate2);
+
+        ItemCandidatesData candidate3 = itemCandidateDataBuilder
+                .withAlgorithm("fixedform")
+                .withSegmentKey("(SBAC_PT)SBAC-MSB-IRP-Perf-MATH-7-Summer-2015-2016")
+                .withSegmentID("SBAC-MSB-IRP-Perf-MATH-7")
+                .withSegmentPosition(3)
+                .build();
+        itemCandidatesData.add(candidate3);
+
+        ItemCandidatesData candidate4 = itemCandidateDataBuilder
+                .withAlgorithm("fixedform")
+                .withSegmentKey("(SBAC_PT)SBAC-MSB-IRP-Perf-MATH-11-Summer-2015-2016")
+                .withSegmentID("SBAC-MSB-IRP-Perf-MATH-11")
+                .withSegmentPosition(4)
+                .build();
+        itemCandidatesData.add(candidate4);
+
+        return itemCandidatesData;
+    }
+
+    private List<TestSegment> retrieveTestSegmentData () {
+        ArrayList<TestSegment> testSegments = new ArrayList<>();
+
+        TestSegmentBuilder testSegmentBuilder = new TestSegmentBuilder();
+        BlueprintBuilder blueprintBuilder = new BlueprintBuilder();
+        ItemPoolBuilder itemPoolBuilder = new ItemPoolBuilder();
+        ItemGroupBuilder itemGroupBuilder = new ItemGroupBuilder();
+        TestItemBuilder testItemBuilder = new TestItemBuilder();
+
+        ArrayList<TestItem> testItems1 = new ArrayList<>();
+        for(int i = 0; i < 3; i++) {
+            testItems1.add(testItemBuilder.build());
+        }
+        ItemGroup itemGroup1 = itemGroupBuilder
+                .withItems(testItems1)
+                .build();
+        List<ItemGroup> itemGroups1 = new ArrayList<>();
+        itemGroups1.add(itemGroup1);
+        ItemPool itemPool1 = itemPoolBuilder
+                .withItemGroups(itemGroups1)
+                .withItems(testItems1)
+                .withSiblingItems(testItems1)
+                .build();
+
+        TestSegment testSegment1 = testSegmentBuilder.build();
+
+        return testSegments;
+    }
+
+    //region filterItemCandidates tests
+
+    @Test
+    public void filterItemCandidatesWithFixedFormFilterShouldReturnTwoResults() {
```

Comment:
We've been rolling with `should` in front of our tests.  So this would be `shouldFilterItemCandidatesWithFirxedForm...`.  Defer to you if that pattern should be followed here.